core/templates/dev/head/galleries/full_gallery.html

```diff
@@ -0,0 +1,121 @@
+{% extends "base.html" %}
+
+<!-- Gallery page is a special case: we switch the main and sub titles -->
+{% block maintitle %}
+  Gallery
+{% endblock maintitle %}
+
+{% block subtitle %}
+  Oppia
+{% endblock subtitle %}
+
+{% block navbar_breadcrumb %}
+{% endblock navbar_breadcrumb %}
+
+{% block local_top_nav_options %}
+  <search-bar></search-bar>
+{% endblock %}
+
+{% block header_js %}
+  {{ super() }}
+  <script type="text/javascript">
+    GLOBALS.LANGUAGE_CODES_AND_NAMES = JSON.parse(
+      '{{LANGUAGE_CODES_AND_NAMES|js_string}}');
+    GLOBALS.CAROUSEL_SLIDES_CONFIG = JSON.parse('{{CAROUSEL_SLIDES_CONFIG|js_string}}');
+  </script> 
+
+  <style>
+    html, body {
+      background-color: #eee;
+    }
+  </style>
+{% endblock header_js %}
+
+{% block content %}
+  <div ng-controller="Gallery" class="oppia-gallery-container">
```

Comment:
Rename the controller to `FullGallery` so that this doesn't depend on the `Gallery` controller.